### PR TITLE
Added Stack and Queue JavaScript Data Structures

### PR DESCRIPTION
**Pull request by Rajdeepc**
Added Stack and Queue JavaScript Data Structures

### CODE REVIEW

File: JavaScript/JavaScript - Queue/queue.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+// Running the program
+
+let q = new Queue();
+q.enqueue("a");
+q.enqueue("b");
+q.enqueue("c");
+q.print();
+q.dequeue();
+console.log(q.front());
+q.print();
```
Comment (amanmehara) [inline JavaScript/JavaScript - Queue/queue.js:46]: Add newline at end of file.

```suggestion
q.print();

```

File: JavaScript/JavaScript - Stack/stack.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+
+var newStack = new Stack();
+
+
+newStack.push(1);
+newStack.push(2);
+console.log(newStack.peek());
+console.log(newStack.pop());
+console.log(newStack.peek());
```
Comment (amanmehara) [inline JavaScript/JavaScript - Stack/stack.js:54]: Add newline at end of file.

```suggestion
console.log(newStack.peek());

```

File: JavaScript/JavaScript - Queue/queue.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
+let Queue = function () {
```
Comment (amanmehara) [inline JavaScript/JavaScript - Queue/queue.js:1]: Add license header.
With 2020 as the year and your name as copyright owner.

https://www.apache.org/licenses/LICENSE-2.0

File: JavaScript/JavaScript - Stack/stack.js [hunk 4]
```diff
@@ -0,0 +1,54 @@
+// create a stack function with javascript to understand how stack works
```
Comment (amanmehara) [inline JavaScript/JavaScript - Stack/stack.js:1]: Add license header.
With 2020 as the year and your name as copyright owner.

https://www.apache.org/licenses/LICENSE-2.0

**Review by amanmehara — CHANGES_REQUESTED** [review]

File: JavaScript/JavaScript - Queue/queue.js (hunk 3 above)
Comment (Rajdeepc) [reply JavaScript/JavaScript - Queue/queue.js:7]: Added License headers with year and copyright owner name

File: JavaScript/JavaScript - Queue/queue.js (hunk 1 above)
Comment (Rajdeepc) [reply JavaScript/JavaScript - Queue/queue.js:56]: Added new line

File: JavaScript/JavaScript - Stack/stack.js (hunk 2 above)
Comment (Rajdeepc) [reply JavaScript/JavaScript - Stack/stack.js:61]: Added newline

File: JavaScript/JavaScript - Stack/stack.js (hunk 4 above)
Comment (Rajdeepc) [reply JavaScript/JavaScript - Stack/stack.js]: Added License headers with year and copyright owner name

**Comment by Rajdeepc** [comment]
Closing this PR and opening a new one since it is not reflecting in my open PR board in my Hacktoberfest profile

**Comment by amanmehara** [comment]
@Rajdeepc contributing to open source should not only be for the sake of **hacktoberfest**!

It's ok now, as you have already created a new one (#104).
You can always contribute something new! 😀

**Pull request closed without merging**